src/widgets/concept-widget.tsx

```diff
@@ -0,0 +1,148 @@
+import {
+  VDomModel, VDomRenderer
+} from '@jupyterlab/apputils';
+import {INotebookTracker} from '@jupyterlab/notebook';
+
+import * as React from 'react';
+
+import {AllOfUsConfig} from '../config';
+import {Concept} from '../generated';
+import {ConceptsService} from '../services/concepts.service';
+
+import {Observable} from 'rxjs/Rx';
+
+export enum LoadingStatus {
+  UNSET,
+  LOADING,
+  LOADED
+}
+
+export class ConceptsWidgetModel extends VDomModel {
+
+  public initialized: boolean;
+  public loadingStatus = LoadingStatus.UNSET;
+  public query = '';
+  public concepts: Array<Concept>;
+  public selectedConcept: Concept;
+}
+
+export class ConceptsWidgetRenderer extends VDomRenderer<ConceptsWidgetModel> {
+
+
+  constructor(configObservable: Observable<AllOfUsConfig>,
+              private conceptsService: ConceptsService,
+              private notebooks: INotebookTracker) {
+    super();
+
+    this.id = 'allofus-concepts';
+    this.title.label = 'Concepts';
+    this.title.closable = true;
+    configObservable.subscribe((config) => {
+      if (!this.model.initialized) {
+        this.model.initialized = true;
+        this.update();
+      }
+    });
+  }
+
+  protected render(): React.ReactElement<any> {
```

Comment:
I couldn't find documentation for this type. Why `any` here - what is that generic value supposed to represent?